docs/dev/app/auth.md

```diff
@@ -4,10 +4,95 @@ prev: /dev/app/
 
 # Authentication
 
-Basic structure; feel free to deviate.
+## User Auth Flows 
+
+**Log In**
```

Comment:
could we make this a `###` header? I think that helps it stand out more as a subsection of the User Auth Flows section